scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -43,6 +43,22 @@ workflow GvsExtractCallset {
           scatter_count = scatter_count
     }
 
+    call GetBQTableLastModifiedDatetime as fq_cohort_extract_table_datetime {
+        input:
+           data_project = data_project,
+           default_dataset = default_dataset,
+           service_account_json = service_account_json,
+           dataset_table = "fq_cohort_extract_table"
```

Comment:
these aren't the values for the table names... it should be like

"~{fq_cohort_extract_table_prefix}__SAMPLES"
"~{fq_cohort_extract_table_prefix}__DATA"

(see the inputs to ExtractTask on line 53)


Since we're using them in two places, maybe instead make two new variables with those values and feed them into both these tasks as well as ExtractTask